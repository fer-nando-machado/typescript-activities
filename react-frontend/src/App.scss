#App {
  text-align: center;
  color: #2c3e50;

  img {
    max-width: 200px;
  }
}
