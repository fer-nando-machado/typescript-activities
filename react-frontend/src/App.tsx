import Activities from "./components/Activity";
import logo from "./assets/logo.png";
import "./App.scss";

function App() {
  return (
    <div id="App">
      <img src={logo} alt="Get Your Guide" />
      <h1>Activities</h1>
      <Activities />
    </div>
  );
}

export default App;
