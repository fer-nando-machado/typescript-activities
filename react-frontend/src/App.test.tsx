import "@testing-library/react";
import "@testing-library/jest-dom";
import { describe, it, expect } from "vitest";
import { render } from "@testing-library/react";
import App from "./App";

describe("App Component", () => {
  it("renders default App home", async () => {
    const { container } = render(<App />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
