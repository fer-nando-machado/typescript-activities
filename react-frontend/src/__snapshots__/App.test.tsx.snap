// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App Component > renders default App home 1`] = `
<div
  id="App"
>
  <img
    alt="Get Your Guide"
    src="/src/assets/logo.png"
  />
  <h1>
    Activities
  </h1>
  <div
    class="activities__container"
  />
</div>
`;
